include/cudawrappers/cu.hpp

```diff
@@ -576,16 +576,19 @@ class DeviceMemory : public Wrapper<CUdeviceptr> {
     checkCudaCall(cuMemHostGetDevicePointer(&_obj, hostMemory, 0));
   }
 
-  explicit DeviceMemory(const DeviceMemory &other, size_t offset, size_t size)
-      : _size(size) {
-    if (size + offset > other.size()) {
-      throw Error(CUDA_ERROR_INVALID_VALUE);
-    }
-    _obj = reinterpret_cast<CUdeviceptr>(reinterpret_cast<char *>(other._obj) +
-                                         offset);
+  void memset(unsigned char value, size_t size) {
```

Comment:
Restore the code that was removed at the rebase:
```suggestion
  explicit DeviceMemory(const DeviceMemory &other, size_t offset, size_t size)
      : _size(size) {
    if (size + offset > other.size()) {
      throw Error(CUDA_ERROR_INVALID_VALUE);
    }
    _obj = reinterpret_cast<CUdeviceptr>(reinterpret_cast<char *>(other._obj) +
                                         offset);
                                         
  void memset(unsigned char value, size_t size) {
```